tests/devapps/XForms/XForms.UWP/XForms.UWP.csproj

```diff
@@ -19,9 +19,10 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <PackageCertificateKeyFile>XForms.UWP_TemporaryKey.pfx</PackageCertificateKeyFile>
-    <PackageCertificateThumbprint>071A6EDD359052E944E102A002DCF852535EC344</PackageCertificateThumbprint>
+    <PackageCertificateThumbprint>F2CBB35D91927C09FCB5A0D35E502FFFD15318C8</PackageCertificateThumbprint>

```

Comment:
This new .pfx file should also replace old XForms.UWP_TemporaryKey.pfx in this directory.

Reply:
Yes. The pfx is also part of this PR.